Gordon360/Controllers/RecIM/SeriesController.cs

```diff
@@ -128,7 +128,7 @@ public async Task<ActionResult> DeleteSeriesCascade(int seriesID)
         /// </summary>
         /// <param name="seriesID"></param>
         [HttpPost]
-        [Route("schedule/{seriesID}")]
+        [Route("{seriesID}/schedule")]
```

Comment:
@bennettforkner, just making sure. But this is the correct order right? I'm not just making stuff up.

Reply:
Love it